.github/workflows/pr-awesome.yml

```diff
@@ -28,5 +28,10 @@ jobs:
         sqlite3 target_data.db .dump > target_data.dump
       continue-on-error: true
       
-    - name: Show differences
+    - name: Show database differences
       run: diff -a target_data.dump current_data.dump || true
+
+    - name: Synchronize local and cloud databases
```

Comment:
Please don't add such steps to the action. You should create master-awesome and main-awesome and add the backend sync step. Also, the main sync should target to test env, while release should target to prod env.

Reply:
I have made corresponding changes in the new commit, please check it.